packages/Babylonian-Tests.package/BPExampleToTestTest.class/instance/testMethodBody.st

```diff
@@ -0,0 +1,5 @@
+tests
+testMethodBody
+	
+	self assert: '(Thing new) doBla: 2 with: 42'
+		equals: ((BPExampleToTest forExample: (BPExampleToTestTest createMockExample)) buildMethod).
```

Comment:
maybe think about an instance variable saving the mock example and which you init in the class' setup method so you can use it everywhere else without creating it again and again